seumich/templates/seumich/student_list_partial.html

```diff
@@ -17,14 +19,54 @@ <h2 class="list-header">Students</h2>
                 <th scope="col" class="hide-small">
                     <strong class="table-column-name">Student ID</strong>
                 </th>
-                <th scope="col" class="hide-small">
+                <th scope="col" class="hide-small" data-sorter="false">
                     <strong class="table-column-name">Status</strong>
                 </th>
-                <th scope="col" class="hide-small">
+                <th scope="col" class="hide-small" data-sorter="false">
                     <strong class="table-column-name">Cohorts</strong>
                 </th>
             </tr>
         </thead>
+        <tfoot>
+            <tr>
+                <th scope="col" class="hide-small">
+                    <strong class="table-column-name">Name</strong>
+                </th>
+                <th scope="col" class="hide-small">
+                    <strong class="table-column-name">Student ID</strong>
+                </th>
+                <th scope="col" class="hide-small" data-sorter="false">
+                    <strong class="table-column-name">Status</strong>
+                </th>
+                <th scope="col" class="hide-small" data-sorter="false">
+                    <strong class="table-column-name">Cohorts</strong>
+                </th>
+            </tr>
+            <tr>
+                <th colspan="4" class="ts-pager form-inline">
+                    <div class="btn-group btn-group-sm" role="group">
+                        <button type="button" class="btn btn-default first"><span
+                                class="glyphicon glyphicon-step-backward"></span></button>
+                        <button type="button" class="btn btn-default prev"><span
+                                class="glyphicon glyphicon-backward"></span></button>
+                    </div>
+                    <span class="pagedisplay"></span>
+                    <div class="btn-group btn-group-sm" role="group">
+                        <button type="button" class="btn btn-default next"><span
+                                class="glyphicon glyphicon-forward"></span></button>
+                        <button type="button" class="btn btn-default last"><span
+                                class="glyphicon glyphicon-step-forward"></span></button>
+                    </div>
+                    <select class="form-control input-sm pagesize" title="Select page size">
+                        <option selected="selected" value="10">10</option>
+                        <option value="20">20</option>
+                        <option value="30">30</option>
+                        <option value="all">All Rows</option>
+                    </select>
+                    <select class="form-control input-sm pagenum" title="Select page number"></select>
```

Comment:
Need label again for this dropdown. Also, I am not how useful it is for user to select page number directly from the drop down list. 

They've already go the arrows to the left for page navigation.

Reply:
Yeah, probably not very useful unless there are a ton of pages. Even then, maybe not even . . . I don't know. I basically just copied the settings from the [Bootstrap 3 theme example](https://mottie.github.io/tablesorter/docs/example-option-theme-bootstrap-v3.html). But we can modify it to work better for us.